config/config.go

```diff
@@ -208,6 +212,7 @@ func (c *Config) getTiKVConfig() *tikvcfg.Config {
 		Path:                  c.Path,
 		EnableForwarding:      c.EnableForwarding,
 		TxnScope:              c.Labels["zone"],
+		EnableLocalTxn:        c.EnableLocalTxn,
```

Comment:
It should affect the `txn_scope`'s value? where use it?

Reply:
It's in tikv/client-go#93.